app/data_model/session_data.py

```diff
@@ -1,6 +1,18 @@
 class SessionData(object):
 
-    def __init__(self, tx_id, eq_id, form_type, period_str, language_code=None, survey_url=None, ru_name=None, ru_ref=None, submitted_time=None):
+    def __init__(self,
+                 tx_id,
+                 eq_id,
+                 form_type,
+                 period_str,
+                 language_code=None,
+                 survey_url=None,
+                 ru_name=None,
+                 ru_ref=None,
+                 submitted_time=None,
+                 case_id=None,
+                 case_ref=None,
+                 account_service_url=None):
```

Comment:
We'll need to remember to consider forward compatibility for these fields when used by the [session_store](https://github.com/ONSdigital/eq-survey-runner/blob/master/app/data_model/session_store.py#L82)

Reply:
Just chatted about this. Good point! Let's consider that a separate job and get the session data incompatibility fixed before the next deployment.

Reply:
Card raised: https://trello.com/c/mQrMxzDw